server/text_generation_server/models/custom_modeling/flash_mixtral_modeling.py

```diff
@@ -45,6 +43,7 @@
     SpeculativeHead,
     get_linear,
 )
+from text_generation_server.layers.moe import SparseMoELayer
```

Comment:
Can this be imported on ipex?

Reply:
I changed the import in `text_generation_server.layers.moe.unquantized` conditional on IPEX not being used (like the old code).